### ISSUE TITLE
Delete CbeNumber from this openapi

### ISSUE THREAD

**Issue by smals-vea** [belgif/openapi-organization-identifier#1]
@pvdbosch Delete CbeNumber from this openapi

**Comment by pvdbosch** [comment]
Removal can only be done in a openapi-organization version 2.x which should then be maintained in parallel with the 1.x versions; this may not be worthwhile yet.

It's already marked as deprecated in current version however:

```
deprecated: true # Use either EnterpriseNumber or EstablishmentUnitNumber, as two separate properties if needed
```

**Comment by pvdbosch** [comment]
I noticed the version with CbeNumber marked as deprecated was updated in github, but not yet released. I released this as [version 1.0.1](https://github.com/belgif/openapi-organization-identifier/releases/tag/v1.0.1). It will also become available on maven central within the next couple days (after index update).

**Comment by pvdbosch** [comment]
When there's a trigger to create a new major version, all deprecated types are review and probably removed. I'll close this issue because there's nothing to do for now.
